composer.json

```diff
@@ -24,6 +24,9 @@
         "twig/twig": "~1.28|~2.0",
         "psr/log": "~1.0"
     },
+    "conflict": {
+        "phpunit/phpunit": "< 4.8.35 || >=5.0 <=5.4.2"
```

Comment:
should written as `<4.8.35|<=5.4.2,>=5.0` for consistency with what we do elsewhere IMHO